src/main/app/src/app/zaken/zaak-view/zaak-view.component.html

```diff
@@ -126,7 +126,8 @@
                                                           (doOpschorting)="editOpschorting($event)"
                                                           [verlengReden]="zaak.redenVerlenging"
                                                           [verlengDuur]="zaak.duurVerlenging"
-                                                          [readonly]="readonly"
+                                                          [readonly]="!zaak.rechten.open"
```

Comment:
is [readonly]="!zaak.rechten.open || zaak.heropend" niet beter ipv een extra property? of [readonly]="!zaak.rechten.doorlooptijden"